### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vasylcf**
Realization:
- Python_basic_with_numpy
- Logistic_regression_as_a_neural_network

### CODE REVIEW

File: Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb [hunk 1]
```diff
@@ -853,19 +971,25 @@
     "    \"\"\"\n",
     "    \n",
     "    ### START CODE HERE ### (≈ 1 line of code)\n",
-    "    loss = None\n",
+    "    loss = np.sum(abs(y-yhat))\n",
```
Comment (egoliuk) [inline Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb]: Возможно тут лучше явно обращаться к функциям numpy
loss = np.sum(np.abs(y-yhat))
потому что "Explicit is better than implicit" https://www.python.org/dev/peps/pep-0020/#id3

Хотя этот код тоже будет правильно работать
https://stackoverflow.com/questions/20953127/why-built-in-functions-like-abs-works-on-numpy-array

File: Week_1/Logistic_regression_as_a_neural_network/Logistic Regression with a Neural Network mindset v5.ipynb [hunk 2]
```diff
@@ -814,18 +938,18 @@
     "    ### START CODE HERE ###\n",
     "    \n",
     "    # initialize parameters with zeros (≈ 1 line of code)\n",
-    "    w, b = None\n",
+    "    w, b = initialize_with_zeros(X_train.shape[0])\n",
     "\n",
     "    # Gradient descent (≈ 1 line of code)\n",
-    "    parameters, grads, costs = None\n",
+    "    parameters, grads, costs = optimize(w, b, X_train, Y_train, num_iterations, learning_rate, print_cost = False)\n",
```
Comment (egoliuk) [inline Week_1/Logistic_regression_as_a_neural_network/Logistic Regression with a Neural Network mindset v5.ipynb]: Не стоит тут хардкодить print_cost = False. Лучше в optimize() пробросить значение print_cost, которое принимает model(), иначе теряется смысл передавать гиперпараметр print_cost в метод model()

Screenshot at 16-11-02

File: Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb (hunk 1 above)
Comment (vasylcf) [reply Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb]: Внес правки и добавил np.abs
Никогда не задумывался, какая разница между np.abs и abs ...

File: Week_1/Logistic_regression_as_a_neural_network/Logistic Regression with a Neural Network mindset v5.ipynb (hunk 2 above)
Comment (vasylcf) [reply Week_1/Logistic_regression_as_a_neural_network/Logistic Regression with a Neural Network mindset v5.ipynb]: Согласен с замечанием, добавил необходимые правки в "model"

**Review by egoliuk — APPROVED** [review]

**Comment by party13** [comment]
great!
